src/include/cache.h

```diff
@@ -199,6 +199,10 @@ struct __wt_cache {
     uint64_t las_insert_count; /* Count of inserts to lookaside */
     uint64_t las_remove_count; /* Count of removes from lookaside */
 
+    /* Original transaction time pair to use for the lookaside inserts */
+    uint64_t orig_txnid_to_las;
```

Comment:
My preference to store this information is at session-level unless we have a compelling reason to make this global.

Reply:
Moved to `WT_SESSION_IMPL` structure.